simulation_parameters/SimulationParameters.cs

```diff
@@ -222,34 +226,69 @@ public GameCredits GetCredits()
         return gameCredits;
     }
 
-    public OrganelleDefinition GetRandomProkaryoticOrganelle(Random random)
+    public DifficultyPreset GetDifficultyPreset(string name)
+    {
+        return difficultyPresets[name];
+    }
+
+    public DifficultyPreset GetDifficultyPresetByIndex(int index)
+    {
+        return difficultyPresets.Values.Where(p => p.Index == index).First();
```

Comment:
```suggestion
        return difficultyPresets.Values.First(p => p.Index == index);
```
I think that's valid LINQ.